modules/microshift-oc-mirror-creating-imageset-config.adoc

```diff
@@ -1,92 +1,62 @@
 ----
 kind: ImageSetConfiguration
-apiVersion: mirror.openshift.io/v1alpha2
-storageConfig: <1>
-  registry:
-    imageURL: <storage_backend> <2>
-    skipTLS: false
+apiVersion: mirror.openshift.io/v2alpha1
 mirror:
   operators:
-  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.20 <3>
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v{ocp-version} <1>
     packages:
-    - name: amq-broker-rhel8 <4>
+    - name: amq-broker-rhel9 <2>
       channels:
-      - name: 7.11.x <5>
+      - name: 7.13.x <3>
+  additionalImages: <4>
+   - name: quay.io/rh_ee_aguidi/multi-platform-container:latest
+   - name: quay.io/rh_ee_aguidi/empty-image:latest
 ----
-<1> Set the backend location where the image set metadata is saved. This location can be a registry or local directory. It is required to specify `storageConfig` values.
-<2> Set the registry URL for the storage backend, such as `<example.com/mirror/oc-mirror-metadata`.
-<3> Set the Operator catalog to retrieve images from.
-<4> Specify the Operator packages to include in the image set. Remove this field to retrieve all packages in the catalog.
-<5> Specify only certain channels of the Operator packages to include in the image set. You must always include the default channel for the Operator package even if you do not use the bundles in that channel. You can find the default channel by running the following command: `oc mirror list operators --catalog=<catalog_name> --package=<package_name>`.
+<1> Set the Operator catalog to retrieve images from.
+<2> Specify the Operator packages to include in the image set. Remove this field to retrieve all packages in the catalog.
+<3> Specify only certain channels of the Operator packages to include in the image set. You must always include the default channel for the Operator package even if you do not use the bundles in that channel. You can find the default channel by running the following command: `oc mirror list operators --catalog=<catalog_name> --package=<package_name>`.
+<4> Specify any additional images to include in the image set. If you do not need to specify additional images, delete this field.
++
+--
+[IMPORTANT]
+====
+The `platform` field, related fields, and Helm are not supported by {microshift-short}.
+====
+--
 
-. Save the updated file.
+. Save the updated file as `imageset-config.yaml`.
+//According to what I see here this file is called `isc.yaml`, https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/disconnected_environments/mirroring-in-disconnected-environments#oc-mirror-migration-differences_oc-mirror-migration-v1-to-v2
+//But here the docs are showing `imageset-config.yaml`, so the oc mirror docs are inconsistent; https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/disconnected_environments/mirroring-in-disconnected-environments#oc-mirror-about-cache-and-workspace-dirs_about-installing-oc-mirror-v2
+//Q: do we need to save this to the workspace directory?
```

Comment:
may be the naming conventions needs to be changed in the doc but both `imageset-config.yaml` and `isc.yaml` are both the same and they can reside in any directory(we do not need to specify i believe) as this is just an input file to the oc-mirror command.